### PR TITLE
Header implementado

### PR DESCRIPTION
**Pull request by CauaMarcus**

### CODE REVIEW

File: furniro-front-end/src/app/body/Body.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import Background from "./images/background.png";
+const Body = () => {
+  return;
+};
+export default Body;
```
Comment (DiegoAndradeD) [inline furniro-front-end/src/app/body/Body.tsx]: Do not use body as a name for page or component, it should be divided in sections. As to, if you are now going to make the banner for the home page, you should name it for what it is. And it should be in the module Home

File: furniro-front-end/src/app/header/Index.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import HeaderButtons from "./components/HeaderButtons";
+import Header from "./components/Header";
+export { HeaderButtons, Header };
```
Comment (DiegoAndradeD) [inline furniro-front-end/src/app/header/Index.tsx]: Try to separate the imports by adding comments describing what they are, like // Components, for components

File: furniro-front-end/src/app/header/components/HeaderButtons.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    },
+    {
+      src: Favorites,
+      alt: "Favorites",
+    },
+    {
+      src: Cart,
+      alt: "Cart",
+    },
+  ];
```
Comment (DiegoAndradeD) [inline furniro-front-end/src/app/header/components/HeaderButtons.tsx]: Create a data folder inside the Header module, create a data file and export it from there, you don't need to use this kind of code directly in your components

File: furniro-front-end/src/app/header/components/HeaderButtons.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      <div className="flex gap-4">
+        {images.map((image, index) => (
+          <button key={index}>
+            <Image className="size-6" src={image.src} alt={image.alt} />
+          </button>
+        ))}
+      </div>
+    </div>
+  );
+};
```
Comment (DiegoAndradeD) [inline furniro-front-end/src/app/header/components/HeaderButtons.tsx]: You could just make everything a buttons, and make they attributes optional, so then you could have buttons with only text or only images

File: furniro-front-end/src/app/page.tsx [hunk 5]
```diff
@@ -1,113 +1,4 @@
...
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50 text-balance`}>
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
-      </div>
-    </main>
-  );
+  return <Header />;
```
Comment (DiegoAndradeD) [inline furniro-front-end/src/app/page.tsx]: Header should go into the layout as it will be present in the whole aplication

**Review by DiegoAndradeD — COMMENTED** [review]
Read comments

File: furniro-front-end/src/app/header/components/HeaderButtons.tsx (hunk 3 above)
Comment (CauaMarcus) [reply furniro-front-end/src/app/header/components/HeaderButtons.tsx]: Data Folder created inside the module, HeaderData.tsx file created and exported.

File: furniro-front-end/src/app/body/Body.tsx (hunk 1 above)
Comment (CauaMarcus) [reply furniro-front-end/src/app/body/Body.tsx]: A module with the name Commons with the modules components and images inside was created, with the file HomeBanner.tsx inside the module components and a background.png image inside the module images.

File: furniro-front-end/src/app/header/Index.tsx (hunk 2 above)
Comment (CauaMarcus) [reply furniro-front-end/src/app/header/Index.tsx]: I added the comments, them being:

// Header components
import Header from "./components/Header";

and

// Buttons and Icons
import HeaderButtons from "./components/HeaderButtons";

File: furniro-front-end/src/app/page.tsx (hunk 5 above)
Comment (CauaMarcus) [reply furniro-front-end/src/app/page.tsx]: Header is now in the layout.

File: furniro-front-end/src/app/Commons/components/HomeBanner.tsx [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import { backgroundData } from "../data/HomeData";
+const HomeBanner = () => {
+  return (
+    <div>
+      <img className="w-full h-full" src="backgroundData" alt="" />
```
Comment (DiegoAndradeD) [inline furniro-front-end/src/app/Commons/components/HomeBanner.tsx]: Image cannot go without alt and in nextjs you should use the Image component from next/image for images

File: furniro-front-end/src/app/Commons/data/HomeData.tsx [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  src: string | StaticImageData;
+  alt: string;
+}
+
+export const backgroundData = () => {
+  {
+    src: HomeBackground;
+    alt: "Background";
+  }
+};
```
Comment (DiegoAndradeD) [inline furniro-front-end/src/app/Commons/data/HomeData.tsx]: You dont need to have a data for a simple image in a single component, just use it directly in the component and it doesn't have to have props for it

File: furniro-front-end/src/app/header/Index.ts [hunk 8]
```diff
@@ -0,0 +1,6 @@
+// Header components
+import Header from "./components/Header";
+// Buttons and Icons
+import HeaderButtons from "./components/HeaderButtons";
+
+export { HeaderButtons, Header };
```
Comment (DiegoAndradeD) [inline furniro-front-end/src/app/header/Index.ts]: I meant like this: 
// Components
import Header from "./components/Header";
import HeaderButtons from "./components/HeaderButtons";

**Review by DiegoAndradeD — COMMENTED** [review]
Read comments

File: furniro-front-end/src/app/Commons/components/HomeBanner.tsx (hunk 6 above)
Comment (CauaMarcus) [reply furniro-front-end/src/app/Commons/components/HomeBanner.tsx]: Resolved

**Pull request merged**